django/contrib/admin/checks.py

```diff
@@ -18,9 +18,13 @@
 
 
 def check_admin_app(**kwargs):
-    from django.contrib.admin.sites import system_check_errors
+    from django.contrib.admin.sites import all_sites
```

Comment:
I think the correct implementation would only run checks against `ModelAdmin` and inlines attached to models which `_meta.apps` is `in app_configs` or `_meta.apps is django.apps.apps` if not provided.